.github/workflows/list_likelihoods.yml

```
@@ -0,0 +1,27 @@
+name: List Public Likelihoods
+
+on:
+  push:
+  pull_request:
+  # Run daily at 0:01 UTC
+  schedule:
+  - cron:  '1 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  atlas:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

```suggestion
    - uses: actions/checkout@v3

    - name: Set up Python
      uses: actions/setup-python@v4
      with:
        python-version: "3.x"
```

We should use the latest actions to avoid having dependabot open PRs to yell at us. We can also just make it easy with the latest Python 3 that GitHub Actions supports as `selenium` `webdriver_manager` wheels are `*py3-none-any.whl` wheels.